src/App.tsx

```diff
@@ -1,10 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export const App: React.FC = () => {
+  const [pressedKey, setPressedKey] = useState('');
+
+  useEffect(() => {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      setPressedKey(event.key);
+    });
+
+    return () => {
+      document.removeEventListener('keyup', (event: KeyboardEvent) => {
+        setPressedKey(event.key);
+      });
+    };
+  }, []);
```

Comment:
```suggestion
  }, [pressedKey]);
```

Reply:
actually no need for a `pressedKey` dependency
because we just set the event listener which will handle that until component unmounts